### ISSUE TITLE
Cannot see Web Vitals in Vercel project

### ISSUE THREAD

**Issue by enricoros** [axiomhq/next-axiom#17]
Hello, I love the Vercel integration! I followed the instructions but cannot get the web vitals metrics on the dashboard (FCP and all).

Added the following to `pages/_app.tsx`
```
export { reportWebVitals } from 'next-axiom';
```

Wrapped next.config.js:
![image](https://user-images.githubusercontent.com/32999/176975235-6b4beec5-fd7f-4b76-bd03-a24de39c9057.png)

Vercel logs work, as well as Axiom log functions (I see the output in the stream as a 'message').

However I don't see WebVitals metrics.

The frontend is sending them to Vercel:
![image](https://user-images.githubusercontent.com/32999/176975433-970dbbae-66ec-4164-a2a7-8d8cfa4c2cf0.png)

Is this expected?

**Comment by enricoros** [comment]
I just started receiving web-vitals data. Turns out only the main branch of a repository will send out the data, other branches will not.

Would update the doc adding a couple of info that could be useful:
 1. Web Vitals to Axiom is in addition to the web-vitals captured by Vercel delopments (data sent to 2 different websites)
 2. Only the production branch in Vercel will get the vitals. Other branches will still get logs, but no vitals.

**Comment by enricoros** [comment]
I just started receiving web-vitals data. Turns out only the main branch of a repository will send out the data, other branches will not.

Would update the doc adding a couple of info that could be useful:

Web Vitals to Axiom is in addition to the web-vitals captured by Vercel delopments (data sent to 2 different websites)
Only the production branch in Vercel will get the vitals. Other branches will still get logs, but no vitals.
